app/views/content/life-as-a-teacher/pay-and-benefits/bens-favourite-things-about-teaching/_header.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= render Content::LandingHeroComponent.new(
+  title: @front_matter ["title"],
```

Comment:
Remove spaces after `@front_matter`to make lines 2 and 3 consistent with 4 and 5